src/app/components/Banner/Banner.js

```diff
@@ -0,0 +1,19 @@
+import cn from './Banner.module.scss';
+
+const APPLICATION_FORM = 'https://forms.gle/e8zAUSGniWopQDNP9';
+
+function Banner() {
+	return (
+		<div className={cn.banner}>
+			<p>
+				Interested in becoming part of Design at UCI&apos;s Board? Submit a{' '}
+				<a className='color ultraviolet' href={APPLICATION_FORM}>
```

Comment:
also can we make the link color blue - all of our links use blue.

Reply:
the issue is for this silver background, the usual blue does not have sufficient contrast ratio (would be 2.34)

Reply:
if it were up to me, i'd say ignore the contrast ratio - if I had to put more work into it, I'd say use a lighter background instead then - to me, the color of the link staying the same is pretty important but I don't feel strong enough to definitively block a pr for it especially since it's leaving in a week.

Reply:
Color scheme was inverted in 1f2db52, approved by Carly and Meghna.